styles/prosilver/theme/pastebin.css

```diff
@@ -25,21 +25,27 @@
 .mini-panel {
 	display: block;
 	padding: 1em ;
-	margin: 1em;
-	
-	
+	margin-top: 1em;
+	margin-bottom: 1em;	
 }
+
 #content {
-	width: 70%;
+	width: 100%;
 }
+
 #extras {
-	float: right;
-	width: 30%;
+	width: 100%;
 }
 
 .mini-panel h3 {
 	margin-top: 0px;
 }
+
 .mini-panel ul {
 	padding-left: 1em;
+}
+
+#copy_paste.inputbox{
+	width: 99%;
+	max-width: 99%;
 }
```

Comment:
Ansich nichts. Ihr Fehlen habe ja auch ich "verbrochen". Das sorgt lediglich dafür, dass in diffs die letzte Zeile nicht als modifiziert erkannt wird, wenn eigentlich nur was an die Datei angehängt wurde. Siehe z.B. https://area51.phpbb.com/docs/31x/coding-guidelines.html